uma/__tests__/test_uma.py

```diff
@@ -1302,6 +1304,203 @@ def test_uma_invoice_signature() -> None:
     assert invoice.signature is not None
 
 
+def test_settlement_options_in_lnurlp_response() -> None:
+    from uma.protocol.settlement import SettlementAsset, SettlementOption
+
+    sender_private_key = generate_key()
+    receiver_private_key = generate_key()
+    receiver_address = "bob@vasp2.com"
+    lnurlp_request_url = create_uma_lnurlp_request_url(
+        signing_private_key=sender_private_key.secret,
+        receiver_address=receiver_address,
+        sender_vasp_domain="vasp1.com",
+        is_subject_to_travel_rule=True,
+    )
+    lnurlp_request = parse_lnurlp_request(lnurlp_request_url)
+    settlement_options = [
+        SettlementOption(
+            settlement_layer="ln",
+            assets=[
+                SettlementAsset(
+                    identifier="BTC",
+                    multipliers={
+                        "USD": 34150.0,
+                        "PHP": 5678.0,
+                    },
+                )
+            ],
+        ),
+        SettlementOption(
+            settlement_layer="spark",
+            assets=[
+                SettlementAsset(
+                    identifier="btkn1...",
+                    multipliers={
+                        "USD": 100.0,
+                    },
+                )
+            ],
+        ),
+    ]
+    metadata = _create_metadata()
+    callback = "https://vasp2.com/api/lnurl/payreq/$bob"
+    min_sendable_sats = 1
+    max_sendable_sats = 10_000_000
+    payer_data_options = create_counterparty_data_options(
+        {"name": False, "email": False}
+    )
+    currencies = [
+        Currency(
+            code="USD",
+            name="US Dollar",
+            symbol="$",
+            millisatoshi_per_unit=34_150,
```

Comment:
@shreyav I just noticed that this field is still awkward on currency objects at the top level until we make the next breaking version change. I guess it doesn't really matter what you put here if you don't support lightning because it'll fail the payreq anyway?